Frontend/DinoTracker/tests/unit/controllers/login-test.ts

```diff
@@ -0,0 +1,39 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import { click, currentURL, visit } from '@ember/test-helpers';
+import Service from '@ember/service';
+
+class MockApolloRepositoryService extends Service
```

Comment:
Don't worry about this holding up this PR but if and When you add other tests that also need this, you should move it to a separate file so it can be re-used. The file would look like this:

Frontend/DinoTracker/tests/test-utils/mock-apollo-service.ts
```
export default class MockApolloRepositoryService extends Service {
  login(variables: { username: string, password: string })
  {
    return variables.username == "user1" && variables.password == "1234";
  }
}
```

Then in the file that uses it:
```
import MockApolloRepositoryService from 'dinotracker/tests/test-utils/mock-apollo-repository'

test('...', async function() {
    this.owner.register('service:apolloRepository', MockApolloRepositoryService);
});
```